source/standalone/tutorials/04_sensors/benchmark_cameras.py

```diff
@@ -0,0 +1,667 @@
+        elif prim_type == "Cone":
+            shape_cfg = sim_utils.ConeCfg(radius=0.1, height=0.25, **common_properties)
+        elif prim_type == "Cylinder":
+            shape_cfg = sim_utils.CylinderCfg(radius=0.25, height=0.25, **common_properties)
+        # Rigid Object
+        obj_cfg = RigidObjectCfg(
+            prim_path=f"/World/Objects/Obj_{i:02d}",
+            spawn=shape_cfg,
+            init_state=RigidObjectCfg.InitialStateCfg(pos=position),
+        )
+        scene_entities[f"rigid_object{i}"] = RigidObject(cfg=obj_cfg)
+
+    # Sensors
+    standard_camera = create_cameras(
+        num_cams=num_standard_cams, data_types=standard_camera_replicators, height=height, width=width
+    )
+    tiled_camera = create_tiled_cameras(
+        num_cams=num_tiled_cams, data_types=tiled_camera_replicators, height=height, width=width
+    )
+    ray_caster_camera = create_ray_caster_cameras(
+        num_cams=num_ray_caster_cams,
+        data_types=ray_caster_camera_replicators,
+        mesh_prim_paths=mesh_prim_paths,
+        height=height,
+        width=width,
+    )
+    # return the scene information
+    if tiled_camera is not None:
+        scene_entities["tiled_camera"] = tiled_camera
+    if standard_camera is not None:
+        scene_entities["standard_camera"] = standard_camera
+    if ray_caster_camera is not None:
+        scene_entities["ray_caster_camera"] = ray_caster_camera
+    return scene_entities
+
+
+def plot_images(
+    images: dict, cols: int = 4, cmap: str = "gray", title_prefix: str = "Image", save_name: str | None = None
+):
+    """Plot the provided images in a grid, labelling each one with the relevant info
+
+    Args:
+        images: the images, where their label is key, and the image is value
+        cols: how many columns to use in the grid. Defaults to 4.
+        cmap: what color map to use. Defaults to "gray".
+        title_prefix: what prefix to prepend to image titles. Defaults to "Image".
+        save_name: Filename to save the image to. Defaults to None.
+    """
+    total_images = sum(image_tensor.shape[0] for image_tensor in images.values())
+
+    # Determine the grid size (rows and columns)
+    cols = min(total_images, cols)
+    rows = (total_images + cols - 1) // cols
+
+    fig, axs = plt.subplots(rows, cols, figsize=(5 * cols, 5 * rows))
+    axs = axs.flatten()  # Flatten the 2D array of axes for easier iteration
+
+    img_idx = 0
+    for key, image_tensor in images.items():
+        for idx, image in enumerate(image_tensor):
+            ax = axs[img_idx]
+            ax.imshow(image.cpu().numpy(), cmap=cmap)
+            ax.set_title(f"{key} - {title_prefix} {idx}")
+            ax.axis("off")  # Hide axis for cleaner visualization
+            img_idx += 1
+
+    # Hide any remaining empty subplots
+    for ax in axs[img_idx:]:
+        ax.axis("off")
+
+    if save_name:
+        plt.savefig(save_name)
+        plt.close()  # Close the figure to free up memory
```

Comment:
better to go around the entire matplotlib implementation and use directly replicator for it, as done here https://github.com/isaac-sim/IsaacLab/blob/a25b994cb4835dc156a5e3d8675b8e88c21b2408/source/extensions/omni.isaac.lab/test/sensors/test_camera.py#L467

Reply:
Opting to remove visualization - I think the viewer could do a similar operation with other existing scripts or the isaac viewport